src/tasks/AppleAppBuilder/Templates/main-simple.m

```diff
@@ -48,7 +48,9 @@ - (void)viewDidLoad {
     [button addTarget:self action:@selector(buttonClicked:) forControlEvents:UIControlEventTouchUpInside];
     [button setFrame:CGRectMake(50, 300, 200, 50)];
     [button setTitle:@"Click me (wire me up)" forState:UIControlStateNormal];
+#if !USE_NATIVE_AOT
     [button setExclusiveTouch:YES];
+#endif
```

Comment:
Is this tvOS specific? Should we use `#if !TARGET_OS_TV` or drop it entirely?

Reply:
Yes, it is not supported on tvOS. I suggest we drop it entirely as it shouldn't change the functionality but could simplify the code.